release/ray_release/scripts/ray_bisect.py

```diff
@@ -28,30 +28,37 @@ def main(test_name: str, passing_commit: str, failing_commit: str) -> None:
         )
         return
     commit_lists = _get_commit_lists(passing_commit, failing_commit)
-    blamed_commit = _bisect(test, commit_lists)
+    blamed_commit = _bisect(test, commit_lists, 3)
     logger.info(f"Blamed commit found for test {test_name}: {blamed_commit}")
 
 
-def _bisect(test: Test, commit_list: List[str]) -> str:
+def _bisect(test: Test, commit_list: List[str], concurrency: int) -> str:
```

Comment:
what is the reasonable range for `concurrency`? like can it be `0`?

maybe add a check for the input (and raise `ValueError` if invalid)?